### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by NaiRobley**
additional data, new files, additional tests

### CODE REVIEW

File: input_files/livingspaces.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Mara:Jane Doe:John Doe:Just AnotherDoe
+Tsavo:Cersei Lannister:Tyrion Lannisater
+Athi:Jon Snow:Arya Stark:Sansa Stark:Bran Stark
+Tana:Daenerys Targaryen
+Lamu:Ned Stark:Robert Baratheon:Jeoffrey Baratheon
```
Comment (Lemmah) [inline input_files/livingspaces.txt]: Good work...you're on task 3! ...you got real cool coding pace!!!

File: system/dojo.py [hunk 2]
```diff
@@ -1,9 +1,22 @@
+import os
+
+
 class Dojo(object):
     def __init__(self):
-        # self.offices = ['Blue', 'Black', 'Yellow', 'Green']
-        # self.living_spaces = ['Tsavo', 'Amboseli', 'Mara']
-        self.offices = {'blue': ['Robley Gori', 'Just AnotherGuy', 'Just AnotherLady']}
-        self.living_spaces = {'yellow': ['Jane Doe', 'John Doe', 'Just AnotherDoe']}
+        self.offices = {'Blue': ['Robley Gori', 'Just AnotherGuy', 'Just AnotherLady'],
```
Comment (Lemmah) [inline system/dojo.py]: Why hard-code?

Comment (NaiRobley) [inline system/dojo.py]: Current working version. Working on reading and writing from file and it's not ready for develop

File: input_files/livingspaces.txt (hunk 1 above)
Comment (NaiRobley) [inline input_files/livingspaces.txt]: Now i just have to read and write properly. That's where the task is

**Pull request merged**